gulpfile.js

```diff
@@ -36,6 +36,8 @@ var merge = require('merge-stream');
 var zip = require('gulp-zip');
 var webpack2 = require('webpack');
 var webpackStream = require('webpack-stream');
+var istanbul = require('gulp-istanbul');
+// var inject = require('gulp-inject');
```

Comment:
Delete unused code instead of commenting it out.

Reply:
done